Codeinwp/otter-blocks
# Better placeholders for content blocks

Closes #1439.


### Summary

- Changes default content in **Accordion**, **Icon List** and **Flip** blocks.
- Adds a custom appender button for the accordion.
- Changed the default style of the Accordion block to `Boxed`
### Screenshots 
<img width="400" alt="Screenshot 2023-02-06 at 23 13 39" src="https://user-images.githubusercontent.com/39873395/217089411-793de3a8-82d5-4133-ac01-51c247a7a0cc.png">

----

### Test instructions


- For the Accordion and Icon list, the content is set as actual text, and for the Flip card, the new content is just a placeholder.
- When adding an Accordion, the default set style should be the `Boxed` one.
- When adding a new Accordion item or a new Icon list item, the title of the element should be set depending on the number of existing elements: if there ar 2 existing item, the third one will be named by default `Accordion item 3` or `List tem 3`



---- 

### Checklist before the final review
